### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

File: pcweb/components/navbar.py [hunk 1]
```diff
@@ -296,6 +296,12 @@ def navbar(sidebar: rx.Component = None) -> rx.Component:
                         style=styles.NAV_TEXT_STYLE,
                         display=["none", "none", "none", "flex", "flex", "flex"],
                     ),
+                    rx.link(
```
Comment (picklelo) [inline pcweb/components/navbar.py:299]: Should this go under the drop down menu, or you think it should be on the main level?

File: pcweb/pages/changelog/changelog.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    return rx.vstack(
+        change(
+            "2021-08-01",
+            "v0.2.6",
+            "Bug fix to connect to no sql databases.",
+            [
+                "Added the step prop to Slider.",
+                "Added support for limits in pagination",
+            ],
+            "/docs",
```
Comment (picklelo) [inline pcweb/pages/changelog/changelog.py:58]: The URL should point to the Github release

File: pcweb/pages/changelog/changelog.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        change(
+            "2021-08-01",
+            "v0.2.5",
+            "Performance improvements + bug fixes.",
+            [
+                "Removing the Node dependency on Windows",
+                "3x performance improvement on the frontend (dev mode should be much snappier now!)",
+                "f-string support for vars",
+            ],
+            "/docs",
```
Comment (picklelo) [inline pcweb/pages/changelog/changelog.py:69]: here also

**Review by picklelo — COMMENTED** [review]
Page looks awesome! Just need to update the links. 

Can we increase the font size a bit / add some more padding around the text, the text isa bit cramped. (Can do in a follow up)

File: pcweb/components/navbar.py (hunk 1 above)
Comment (Alek99) [reply pcweb/components/navbar.py:299]: Main level

**Review by picklelo — APPROVED** [review]

**Pull request merged**